### PR TITLE
Remove invalid repeatable CSS

### PR DESCRIPTION
**Pull request by timja**
See https://github.com/jenkinsci/jenkins/pull/6467#pullrequestreview-940209292 and [JENKINS-14749](https://issues.jenkins.io/browse/JENKINS-14749)

Validated with the Git plugin, all looks good. Because these rules were invalid they would have been ignored,

I also cherry-picked this PR on top of https://github.com/jenkinsci/jenkins/pull/6467#pullrequestreview-940209292 and run

```
mvn clean verify -Dtest=lib.form.FormTest#autocompleteOffByDefault
```

Which showed no warnings anymore, output:

```
[INFO] Running lib.form.FormTest
=== Starting autocompleteOffByDefault(lib.form.FormTest)
   0.040 [id=17]	INFO	o.jvnet.hudson.test.WarExploder#explode: Using jenkins.war resources from /Users/timja/code/jenkins/jenkins/war/target/jenkins
   0.157 [id=17]	INFO	o.jvnet.hudson.test.JenkinsRule#createWebServer: Running on http://localhost:62659/jenkins/
   0.313 [id=31]	INFO	jenkins.InitReactorRunner$1#onAttained: Started initialization
   0.315 [id=30]	INFO	jenkins.InitReactorRunner$1#onAttained: Listed all plugins
   0.834 [id=31]	INFO	jenkins.InitReactorRunner$1#onAttained: Prepared all plugins
   0.844 [id=36]	INFO	jenkins.InitReactorRunner$1#onAttained: Started all plugins
   0.849 [id=34]	INFO	jenkins.InitReactorRunner$1#onAttained: Augmented all extensions
   1.187 [id=38]	INFO	jenkins.InitReactorRunner$1#onAttained: System config loaded
   1.187 [id=30]	INFO	jenkins.InitReactorRunner$1#onAttained: System config adapted
   1.188 [id=33]	INFO	jenkins.InitReactorRunner$1#onAttained: Loaded all jobs
   1.190 [id=33]	INFO	jenkins.InitReactorRunner$1#onAttained: Configuration for all jobs updated
   1.281 [id=30]	INFO	jenkins.InitReactorRunner$1#onAttained: Completed initialization
   3.749 [id=17]	INFO	hudson.lifecycle.Lifecycle#onStatusUpdate: Stopping Jenkins
WARN: The method class org.apache.commons.logging.impl.SLF4JLogFactory#release() was invoked.
WARN: Please see http://www.slf4j.org/codes.html#release for an explanation.
   4.266 [id=17]	INFO	hudson.lifecycle.Lifecycle#onStatusUpdate: Jenkins stopped
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.771 s - in lib.form.FormTest
[INFO]
```

I've also validated that there is no more invalid CSS (at least in the `FormTest`) except for YUI, so we can remove the hacks to ignore, see https://github.com/jenkinsci/jenkins-test-harness/pull/424

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by NotMyFault — APPROVED** [review]

**Review by basil — APPROVED** [review]
This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Comment by daniel-beck** [comment]
Probably caused [JENKINS-72018](https://issues.jenkins.io/browse/JENKINS-72018).

**Pull request merged**
